example/create_add_role.dart

```diff
@@ -18,15 +18,14 @@ void main() async {
   bot.eventsWs.onMessageReceived.listen((IMessageReceivedEvent e) async {
     // Check if message content equals "!embed"
     if (e.message.content == "!role") {
-
-      // Make sure that message was sent in guild not im dm, because we cant add roles in dms
-      if (e.message.guild != null) {
+      // Make sure that the message was sent in a guild and not in a dm, because we cant add roles in dms
+      if (e.message.guild == null) {
         return;
       }
 
       // Creating role with RoleBuilder. We have to cast `e.message` to GuildMessage because we want to access guild property
       // and generic dont have that.
```

Comment:
While you're at it, you can remove these lines as they don't have any meaning now
```suggestion
      // Creating a role with RoleBuilder with a given color.
```